packages/blade/src/components/Card/CardFooter.tsx

```diff
@@ -40,6 +40,15 @@ const _CardFooter = ({ children, testID }: CardFooterProps): React.ReactElement
     ComponentIds.CardFooterTrailing,
   ]);
 
+  const footerChildrensArray = React.Children.toArray(children);
+
+  const baseBoxJustifyContent =
+    footerChildrensArray.length && footerChildrensArray.length === 2
```

Comment:
```suggestion
    footerChildrensArray.length === 2
```

this should be enough here right?

Reply:
=== 2 itself will ignore any falsy values

Reply:
makes sense, thanks